MiniAn/minian_utilities.py

```diff
@@ -342,4 +333,7 @@ def set_advanced_parameters_for_estimate_motion(
         param_func[key] = value
         advanced_parameters[key] = value
 
-    return [param_func, advanced_parameters]
\ No newline at end of file
+    return [param_func, advanced_parameters]
+
+def print_error(error):
+    print("An error occurred:", type(error).__name__, "-", error, flush=True)
```

Comment:
Yes. It will be in the log (it's mostly for use incase something happen)

Also I'm changing it's place.
I'm putting it in the common utils (because I will use it in caiman too)